comment_spell_check/comment_spell_check.py

```diff
@@ -18,22 +18,33 @@
 #
 # ==========================================================================*/
 
+"""spell check the comments in code."""
+
 import sys
 import os
 import fnmatch
 import glob
 import argparse
 import re
+import unicodedata
+import logging
 from pathlib import Path
 from importlib.metadata import version, PackageNotFoundError
 
-from enchant.checker import SpellChecker
-from enchant.tokenize import EmailFilter, URLFilter
-from enchant import Dict
-
 from comment_parser import comment_parser
 
-from comment_spell_check.lib import bibtex_loader
+from spellchecker import SpellChecker
+
+try:
+    # This loads the modules from the installed package
+    from comment_spell_check.utils import bibtex_loader
+    from comment_spell_check.utils import create_checker
+    from comment_spell_check.utils import url_remove
+except ImportError:
```

Comment:
This is a creative. Only the full module path should be used. Please delete the try and except handing.

Reply:
It had to do with running the script via: "python comment_spell_check.py" versus the command line "comment_spell_check" after you've done the pip install.  I couldn't quite figure it out otherwise.